src/MultiSigner/BNAggregate.js

```diff
@@ -0,0 +1,92 @@
+const BNSignerWrapper = require('../GoWrappers/BNSignerWrapper.js')
+const validator = require('../Validator.js');
+/**
+ * BNAggregate
+ * @class BNAggregate
+ */
+class BNAggregate {
+    /**
+     * Creates an instance of BNAggregate.
+     * @param {Object} BNSigner class
+     */
+    constructor(bnSigner) {
+        this.bnSigner = bnSigner
+    }
+
+    async publicKeys(publicKeys) {
+        try {
+            if (!publicKeys || publicKeys.length == 0) {
```

Comment:
JSLint Expected '===' and instead saw '==' 😘

Reply:
I see lint everywhere in the future.